go-selinux/selinux.go

```diff
@@ -12,8 +12,9 @@ const (
 	// Disabled constant to indicate SELinux is disabled
 	Disabled = -1
 
+	MAX_CATEGORY = 1024
```

Comment:
This should be `MaxCategory` or `maxCategory`

Is this intended to be used externally? or only to initialize `DefaultCategoryRange` ? If the latter, could you un-export the const (`maxCategory`)?

If this is meant for external use, and must be exported, could you add a GoDoc comment to this (linters may fail on this otherwise)

Reply:
Yikes, quite the brain fart. 

Fixed.